dev/src/MockGRPCManager.ts

```diff
@@ -72,7 +92,9 @@ export class MockGrpcManager {
     private organizations: Organizations
     private submissions: Submissions
     private templateBenchmarks: GradingBenchmark[]
-
+    // idMap is a map of auto incrementing IDs
+    public idMap: Map<string, number> = new Map<string, number>()
+    public e: typeof Generate = Generate
```

Comment:
Not clear what this `e` is. Perhaps name it with a longer name or add a comment?